diff --git a/scripts/chroot.sh b/scripts/chroot.sh
index 05a318aa..a7a8f6ef 100755
--- a/scripts/chroot.sh
+++ b/scripts/chroot.sh
@@ -1300,7 +1300,7 @@ cat > "${DIR}/cleanup_script.sh" <<-__EOF__
 
 	if [ -f /usr/bin/connmanctl ] ; then
 		rm -rf /etc/resolv.conf.bak || true
-		rm -rf /etc/resolv.conf || true
+		# rm -rf /etc/resolv.conf || true
 		ln -s /run/connman/resolv.conf /etc/resolv.conf
 	fi
 
diff --git a/scripts/install_dependencies.sh b/scripts/install_dependencies.sh
index ebe22455..4e6e1cf2 100755
--- a/scripts/install_dependencies.sh
+++ b/scripts/install_dependencies.sh
@@ -33,7 +33,7 @@ debootstrap_is_installed () {
 		if [ "x${host_arch}" != "xaarch64" ] ; then
 			echo "QEMU is un-reliable, thus no longer supported... Spend some Money and buy a real ARMHF device to run this script."
 			#FIXME: comment out the next line to use QEMU
-			exit 2
+			# exit 2
 			dpkg -l | grep qemu-user-static >/dev/null || deb_pkgs="${deb_pkgs}qemu-user-static "
 			dpkg -l | grep $(dpkg --print-architecture) | grep -v "qemu-" | grep qemu >/dev/null || deb_pkgs="${deb_pkgs}qemu "
 		fi
diff --git a/tools/hwpack/beaglebone.conf b/tools/hwpack/beaglebone.conf
index 7031c3ef..4a4e5dd7 100644
--- a/tools/hwpack/beaglebone.conf
+++ b/tools/hwpack/beaglebone.conf
@@ -32,3 +32,5 @@ SERIAL="ttyO0"
 drm_device_identifier="HDMI-A-1"
 
 conf_eeprom_compare="335"
+
+DISABLE_ETH=0
\ No newline at end of file
diff --git a/tools/setup_sdcard.sh b/tools/setup_sdcard.sh
index ffed9126..786059eb 100755
--- a/tools/setup_sdcard.sh
+++ b/tools/setup_sdcard.sh
@@ -1436,7 +1436,7 @@ populate_rootfs () {
 			if [ "${BTRFS_FSTAB}" ] ; then
 				echo "${rootfs_drive}  /  btrfs  defaults,noatime  0  1" >> ${wfile}
 			else
-				echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro  0  1" >> ${wfile}
+				echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro,usrquota,grpquota 0 1" >> ${wfile}
 			fi
 		fi
 
