diff --git a/tools/setup_sdcard.sh b/tools/setup_sdcard.sh
index 3d0f45db..8ee3a577 100755
--- a/tools/setup_sdcard.sh
+++ b/tools/setup_sdcard.sh
@@ -1324,7 +1324,7 @@ populate_rootfs () {
 			echo "console=${extlinux_console}" >> ${wfile}
 		fi
 
-		cmdline="coherent_pool=1M net.ifnames=0"
+		cmdline="coherent_pool=1M net.ifnames=0 security=apparmor"
 
 		if [ ! "x${loops_per_jiffy}" = "x" ] ; then
 			cmdline="${cmdline} ${loops_per_jiffy}"
@@ -1468,7 +1468,7 @@ populate_rootfs () {
 		if [ "${BTRFS_FSTAB}" ] ; then
 			echo "${rootfs_drive}  /  btrfs  defaults,noatime  0  1" >> ${wfile}
 		else
-			echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro  0  1" >> ${wfile}
+			echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro,usrquota,grpquota  0  1" >> ${wfile}
 		fi
 
 		if [ "x${uboot_efi_mode}" = "xenable" ] ; then
@@ -1798,7 +1798,7 @@ check_dtb_board () {
 
 usage () {
 	echo "usage: sudo $(basename $0) --mmc /dev/sdX --dtb <dev board>"
-	#tabed to match 
+	#tabed to match
 		cat <<-__EOF__
 			-----------------------------
 			Bugs email: "bugs at rcn-ee.com"
