--- setup_sdcard_org.sh	2020-07-22 13:33:50.000000000 -0500
+++ setup_sdcard.sh	2020-07-22 13:34:01.000000000 -0500
@@ -1301,7 +1301,7 @@
 		fi
 	fi
 
-	cmdline="coherent_pool=1M net.ifnames=0"
+	cmdline="coherent_pool=1M net.ifnames=0 security=apparmor"
 
 	if [ ! "x${rng_core}" = "x" ] ; then
 		cmdline="${cmdline} ${rng_core}"
@@ -1435,7 +1435,7 @@
 			if [ "${BTRFS_FSTAB}" ] ; then
 				echo "${rootfs_drive}  /  btrfs  defaults,noatime  0  1" >> ${wfile}
 			else
-				echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro  0  1" >> ${wfile}
+				echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro,usrquota,grpquota  0  1" >> ${wfile}
 			fi
 		fi
 
@@ -1457,29 +1457,8 @@
 				echo "" >> ${wfile}
 				echo "# The primary network interface" >> ${wfile}
 
-				if [ "${DISABLE_ETH}" ] ; then
-					echo "#auto eth0" >> ${wfile}
-					echo "#iface eth0 inet dhcp" >> ${wfile}
-				else
-					echo "auto eth0"  >> ${wfile}
-					echo "iface eth0 inet dhcp" >> ${wfile}
-				fi
-
-				#if we have systemd & wicd-gtk, disable eth0 in /etc/network/interfaces
-				if [ -f ${TEMPDIR}/disk/lib/systemd/systemd ] ; then
-					if [ -f ${TEMPDIR}/disk/usr/bin/wicd-gtk ] ; then
-						sed -i 's/auto eth0/#auto eth0/g' ${wfile}
-						sed -i 's/allow-hotplug eth0/#allow-hotplug eth0/g' ${wfile}
-						sed -i 's/iface eth0 inet dhcp/#iface eth0 inet dhcp/g' ${wfile}
-					fi
-				fi
-
-				#if we have connman, disable eth0 in /etc/network/interfaces
-				if [ -f ${TEMPDIR}/disk/etc/init.d/connman ] ; then
-					sed -i 's/auto eth0/#auto eth0/g' ${wfile}
-					sed -i 's/allow-hotplug eth0/#allow-hotplug eth0/g' ${wfile}
-					sed -i 's/iface eth0 inet dhcp/#iface eth0 inet dhcp/g' ${wfile}
-				fi
+				echo "auto eth0" >> ${wfile}
+				echo "iface eth0 inet dhcp" >> ${wfile}
 
 				echo "# Example to keep MAC address between reboots" >> ${wfile}
 				echo "#hwaddress ether DE:AD:BE:EF:CA:FE" >> ${wfile}
