diff --git a/scripts/install_dependencies.sh b/scripts/install_dependencies.sh
index ebe22455..4e6e1cf2 100755
--- a/scripts/install_dependencies.sh
+++ b/scripts/install_dependencies.sh
@@ -33,7 +33,7 @@ debootstrap_is_installed () {
 		if [ "x${host_arch}" != "xaarch64" ] ; then
 			echo "QEMU is un-reliable, thus no longer supported... Spend some Money and buy a real ARMHF device to run this script."
 			#FIXME: comment out the next line to use QEMU
-			exit 2
+			# exit 2
 			dpkg -l | grep qemu-user-static >/dev/null || deb_pkgs="${deb_pkgs}qemu-user-static "
 			dpkg -l | grep $(dpkg --print-architecture) | grep -v "qemu-" | grep qemu >/dev/null || deb_pkgs="${deb_pkgs}qemu "
 		fi
