diff --git a/scripts/chroot.sh b/scripts/chroot.sh
index 4d4d8773..529ac529 100755
--- a/scripts/chroot.sh
+++ b/scripts/chroot.sh
@@ -155,19 +155,19 @@ check_defines () {
 	if [ ! "x${deb_desktop_pkgs}" = "x" ] ; then
 		deb_desktop_pkgs="$(echo ${deb_desktop_pkgs} | sed 's/,/ /g' | sed 's/\t/,/g')"
 	fi
- 
+
     if [ ! "x${deb_desktop_additiona_pkgs}" = "x" ] ; then
         deb_desktop_pkgs="$(echo ${deb_desktop_pkgs} | sed 's/,/ /g' | sed 's/\t/,/g')"
     fi
-    
+
     if [ ! "x${deb_desktop_application_pkgs}" = "x" ] ; then
         deb_desktop_pkgs="$(echo ${deb_desktop_pkgs} | sed 's/,/ /g' | sed 's/\t/,/g')"
     fi
-    
+
     if [ ! "x${deb_desktop_additional_pkgs}" = "x" ] ; then
         deb_desktop_additional_pkgs="$(echo ${deb_desktop_additional_pkgs} | sed 's/,/ /g' | sed 's/\t/,/g')"
     fi
-    
+
     if [ ! "x${deb_desktop_application_pkgs}" = "x" ] ; then
         deb_desktop_application_pkgs="$(echo ${deb_desktop_application_pkgs} | sed 's/,/ /g' | sed 's/\t/,/g')"
     fi
@@ -444,6 +444,11 @@ bookworm|sid)
 	echo "##deb-src http://deb.debian.org/debian ${deb_codename}-updates ${deb_components}" >> ${wfile}
 	echo "" >> ${wfile}
 	;;
+focal)
+	echo "#deb http://deb.debian.org/debian ${deb_codename}-updates ${deb_components}" >> ${wfile}
+	echo "##deb-src http://deb.debian.org/debian ${deb_codename}-updates ${deb_components}" >> ${wfile}
+	echo "" >> ${wfile}
+	;;
 esac
 
 #https://wiki.debian.org/Backports
@@ -738,7 +743,7 @@ cat > "${DIR}/chroot_script.sh" <<-__EOF__
             apt-get update
             apt-get -y install ${deb_desktop_additional_pkgs}
         fi
-        
+
         if [ ! "x${deb_desktop_application_pkgs}" = "x" ] ; then
             #Install the user choosen list.
             echo "Log: (chroot) Installing: ${deb_desktop_application_pkgs}"
diff --git a/tools/setup_sdcard.sh b/tools/setup_sdcard.sh
index 3d0f45db..f64106fd 100755
--- a/tools/setup_sdcard.sh
+++ b/tools/setup_sdcard.sh
@@ -1324,7 +1324,7 @@ populate_rootfs () {
 			echo "console=${extlinux_console}" >> ${wfile}
 		fi
 
-		cmdline="coherent_pool=1M net.ifnames=0"
+		cmdline="coherent_pool=1M net.ifnames=0 security=apparmor"
 
 		if [ ! "x${loops_per_jiffy}" = "x" ] ; then
 			cmdline="${cmdline} ${loops_per_jiffy}"
@@ -1466,7 +1466,7 @@ populate_rootfs () {
 		echo "#" >> ${wfile}
 
 		if [ "${BTRFS_FSTAB}" ] ; then
-			echo "${rootfs_drive}  /  btrfs  defaults,noatime  0  1" >> ${wfile}
+			echo "${rootfs_drive}  /  btrfs  defaults,noatime,usrquota,grpquota  0  1" >> ${wfile}
 		else
 			echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro  0  1" >> ${wfile}
 		fi
@@ -1798,7 +1798,7 @@ check_dtb_board () {
 
 usage () {
 	echo "usage: sudo $(basename $0) --mmc /dev/sdX --dtb <dev board>"
-	#tabed to match 
+	#tabed to match
 		cat <<-__EOF__
 			-----------------------------
 			Bugs email: "bugs at rcn-ee.com"
