diff --git a/tools/setup_sdcard.sh b/tools/setup_sdcard.sh
index a8af8477..121d88dc 100755
--- a/tools/setup_sdcard.sh
+++ b/tools/setup_sdcard.sh
@@ -1368,7 +1368,7 @@ populate_rootfs () {
 		fi
 	fi
 
-	cmdline="coherent_pool=1M net.ifnames=0"
+	cmdline="coherent_pool=1M net.ifnames=0 security=apparmor"
 
 	if [ ! "x${loops_per_jiffy}" = "x" ] ; then
 		cmdline="${cmdline} ${loops_per_jiffy}"
@@ -1510,7 +1510,7 @@ populate_rootfs () {
 			if [ "${BTRFS_FSTAB}" ] ; then
 				echo "${rootfs_drive}  /  btrfs  defaults,noatime  0  1" >> ${wfile}
 			else
-				echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro  0  1" >> ${wfile}
+				echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro,usrquota,grpquota  0  1" >> ${wfile}
 			fi
 		fi
 
@@ -1532,30 +1532,8 @@ populate_rootfs () {
 				echo "" >> ${wfile}
 				echo "# The primary network interface" >> ${wfile}
 
-				if [ "${DISABLE_ETH}" ] ; then
-					echo "#auto eth0" >> ${wfile}
-					echo "#iface eth0 inet dhcp" >> ${wfile}
-				else
-					echo "auto eth0"  >> ${wfile}
-					echo "iface eth0 inet dhcp" >> ${wfile}
-				fi
-
-				#if we have systemd & wicd-gtk, disable eth0 in /etc/network/interfaces
-				if [ -f ${TEMPDIR}/disk/lib/systemd/systemd ] ; then
-					if [ -f ${TEMPDIR}/disk/usr/bin/wicd-gtk ] ; then
-						sed -i 's/auto eth0/#auto eth0/g' ${wfile}
-						sed -i 's/allow-hotplug eth0/#allow-hotplug eth0/g' ${wfile}
-						sed -i 's/iface eth0 inet dhcp/#iface eth0 inet dhcp/g' ${wfile}
-					fi
-				fi
-
-				#if we have connman, disable eth0 in /etc/network/interfaces
-				if [ -f ${TEMPDIR}/disk/etc/init.d/connman ] ; then
-					sed -i 's/auto eth0/#auto eth0/g' ${wfile}
-					sed -i 's/allow-hotplug eth0/#allow-hotplug eth0/g' ${wfile}
-					sed -i 's/iface eth0 inet dhcp/#iface eth0 inet dhcp/g' ${wfile}
-				fi
-
+				echo "auto eth0" >> ${wfile}
+				echo "iface eth0 inet dhcp" >> ${wfile}
 				echo "# Example to keep MAC address between reboots" >> ${wfile}
 				echo "#hwaddress ether DE:AD:BE:EF:CA:FE" >> ${wfile}
 
@@ -1834,7 +1812,7 @@ check_dtb_board () {
 
 usage () {
 	echo "usage: sudo $(basename $0) --mmc /dev/sdX --dtb <dev board>"
-	#tabed to match 
+	#tabed to match
 		cat <<-__EOF__
 			-----------------------------
 			Bugs email: "bugs at rcn-ee.com"
