diff --git a/scripts/chroot.sh b/scripts/chroot.sh
index a96b6e61..434140d4 100755
--- a/scripts/chroot.sh
+++ b/scripts/chroot.sh
@@ -444,6 +444,11 @@ bookworm|sid)
 	echo "##deb-src http://deb.debian.org/debian ${deb_codename}-updates ${deb_components}" >> ${wfile}
 	echo "" >> ${wfile}
 	;;
+focal)
+	echo "deb http://${deb_mirror} ${deb_codename}-updates ${deb_components}" >> ${wfile}
+	echo "#deb-src http://${deb_mirror} ${deb_codename}-updates ${deb_components}" >> ${wfile}
+	echo "" >> ${wfile}
+	;;
 esac
 
 #https://wiki.debian.org/Backports
diff --git a/tools/setup_sdcard.sh b/tools/setup_sdcard.sh
index ce2724c1..0e0f3ebf 100755
--- a/tools/setup_sdcard.sh
+++ b/tools/setup_sdcard.sh
@@ -1227,7 +1227,7 @@ populate_rootfs () {
 			echo "console=${extlinux_console}" >> ${wfile}
 		fi
 
-		cmdline="coherent_pool=1M net.ifnames=0"
+		cmdline="coherent_pool=1M net.ifnames=0 security=apparmor"
 
 		if [ ! "x${loops_per_jiffy}" = "x" ] ; then
 			cmdline="${cmdline} ${loops_per_jiffy}"
@@ -1371,7 +1371,7 @@ populate_rootfs () {
 		if [ "${BTRFS_FSTAB}" ] ; then
 			echo "${rootfs_drive}  /  btrfs  defaults,noatime  0  1" >> ${wfile}
 		else
-			echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro  0  1" >> ${wfile}
+			echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro,usrquota,grpquota  0  1" >> ${wfile}
 		fi
 
 		if [ "x${uboot_efi_mode}" = "xenable" ] ; then
