diff --git a/scripts/chroot.sh b/scripts/chroot.sh
index a96b6e61..3b6aa301 100755
--- a/scripts/chroot.sh
+++ b/scripts/chroot.sh
@@ -444,6 +444,11 @@ bookworm|sid)
 	echo "##deb-src http://deb.debian.org/debian ${deb_codename}-updates ${deb_components}" >> ${wfile}
 	echo "" >> ${wfile}
 	;;
+focal)
+	echo "deb http://${deb_mirror} ${deb_codename}-updates ${deb_components}" >> ${wfile}
+	echo "#deb-src http://${deb_mirror} ${deb_codename}-updates ${deb_components}" >> ${wfile}
+	echo "" >> ${wfile}
+	;;
 esac
 
 #https://wiki.debian.org/Backports
@@ -1100,8 +1105,18 @@ cat > "${DIR}/chroot_script.sh" <<-__EOF__
 		fi
 
 		if [ ! "x${rfs_use_systemdnetworkd}" = "x" ] ; then
-			if [ -f /etc/bbb.io/templates/eth0-DHCP.network ] ; then
-				cp -v /etc/bbb.io/templates/eth0-DHCP.network /etc/systemd/network/eth0.network
+			echo "[Match]" > /etc/systemd/network/eth0.network
+			echo "Name=eth0" >> /etc/systemd/network/eth0.network
+			echo "Type=ether" >> /etc/systemd/network/eth0.network
+			echo "" >>  /etc/systemd/network/eth0.network
+			echo "[Link]" > /etc/systemd/network/eth0.network
+			echo "RequiredForOnline=no" >> /etc/systemd/network/eth0.network
+			echo "" >>  /etc/systemd/network/eth0.network
+			echo "[Network]" >>  /etc/systemd/network/eth0.network
+			echo "DHCP=ipv4" >>  /etc/systemd/network/eth0.network
+
+			if [ -f /lib/systemd/system/systemd-resolved.service ] ; then
+				systemctl disable systemd-resolved.service || true
 			fi
 
 			if [ -f /lib/systemd/system/systemd-networkd.service ] ; then
@@ -1523,7 +1538,8 @@ if [ -f "${tempdir}/etc/dogtag" ] ; then
 fi
 
 #Add Google IPv4 nameservers
-if [ -f "${tempdir}/etc/resolv.conf" ] ; then
+if [ ! -f "${tempdir}/etc/resolv.conf" ] ; then
+	sudo rm ${tempdir}/etc/resolv.conf || true
 	wfile="${tempdir}/etc/resolv.conf"
 	sudo sh -c "echo 'nameserver 8.8.8.8' > ${wfile}"
 	sudo sh -c "echo 'nameserver 8.8.4.4' >> ${wfile}"
diff --git a/tools/setup_sdcard.sh b/tools/setup_sdcard.sh
index ce2724c1..0e0f3ebf 100755
--- a/tools/setup_sdcard.sh
+++ b/tools/setup_sdcard.sh
@@ -1227,7 +1227,7 @@ populate_rootfs () {
 			echo "console=${extlinux_console}" >> ${wfile}
 		fi
 
-		cmdline="coherent_pool=1M net.ifnames=0"
+		cmdline="coherent_pool=1M net.ifnames=0 security=apparmor"
 
 		if [ ! "x${loops_per_jiffy}" = "x" ] ; then
 			cmdline="${cmdline} ${loops_per_jiffy}"
@@ -1371,7 +1371,7 @@ populate_rootfs () {
 		if [ "${BTRFS_FSTAB}" ] ; then
 			echo "${rootfs_drive}  /  btrfs  defaults,noatime  0  1" >> ${wfile}
 		else
-			echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro  0  1" >> ${wfile}
+			echo "${rootfs_drive}  /  ${ROOTFS_TYPE}  noatime,errors=remount-ro,usrquota,grpquota  0  1" >> ${wfile}
 		fi
 
 		if [ "x${uboot_efi_mode}" = "xenable" ] ; then
